package.json

```diff
@@ -306,6 +306,7 @@
     "randomcolor": "^0.5.4",
     "rc": "^1.2.8",
     "react-devtools": "^4.11.0",
+    "react-test-renderer": "^17.0.2",
```

Comment:
is this actually being used? are we not able to accomplish something without this?

Reply:
No this is not needed. I was experimenting with it, but decided against it.